watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_0D.py

```diff
@@ -67,7 +67,7 @@ def test_config():
     m.fs.properties = props.NaClParameterBlock()
     m.fs.unit = OsmoticallyAssistedReverseOsmosis0D(property_package=m.fs.properties)
 
-    assert len(m.fs.unit.config) == 13
+    assert len(m.fs.unit.config) == 14
```

Comment:
I guess one question to consider is whether SKK is applicable to OARO. Never thought about it before myself...

Reply:
In any case, we should have tests to demonstrate successful model creation, scaling, initialization, and solve, following other test examples to make sure nothing breaks.

Reply:
I am still uncertain about this one... if we don't think SKK should be part of OARO, maybe we should disable it for OARO. Otherwise, I guess we can create tests for OARO with SKK,

Reply:
Based on our discussion yesterday, it will be a part of OARO but tests will be added in a separate PR